components/faunadb/actions/read-from-collection/read-from-collection.mjs

```diff
@@ -1,43 +1,40 @@
-// legacy_hash_id: a_poizPB
-import faunadb from "faunadb";
+// import faunadb from "faunadb";
+import faunadb from "../../faunadb.app.mjs";
 
 export default {
   key: "faunadb-read-from-collection",
   name: "Read From FaunaDB Collection",
   description: "Reads all documents from given FaunaDB collection",
-  version: "0.4.1",
+  version: "0.4.2",
   type: "action",
   props: {
-    faunadb: {
-      type: "app",
-      app: "faunadb",
-    },
+    faunadb,
     collectionName: {
-      type: "string",
+      propDefinition: [
+        faunadb,
+        "collections",
+      ],
     },
     documentField: {
+      label: "Documents Field",
+      description: "The field will be returned by query",
       type: "string",
       optional: true,
     },
```

Comment:
Does the API have a way to retrieve the `documentFields`? If so, we should create async options for the `documentField` prop.

Reply:
Don't have a way to retrieve the fields.